docs/content/getting-started/production-environment.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Production Environment
+description: Running OpenFGA in Production Environment
+sidebar_position: 8
+slug: /getting-started/production-environment
+---
+
+import {
+  DocumentationNotice,
+  RelatedSection,
+} from "@components/Docs";
+
+# Running OpenFGA in Production Environment
+
+<DocumentationNotice />
+
+## Preparation
+
+If you have tried OpenFGA locally, decided it works as expected, and now want to deploy it to production, we recommend the following preparation:
+
+- Update the configuration as follows:
+  - [Configure authentication](./setup-openfga.mdx#configuring-authentication) on the server.
+  - Enable TLS on the server.
+  - Enable production logging by changing the `log.format` in the [config.yaml](#configuring-the-server) configuration file to `json`.
+  - [Disable Playground](./setup-openfga.mdx#playground).
+- Add a rate limiter interceptor (for example, [in the server configuration](https://pkg.go.dev/github.com/openfga/openfga/server#Config)) that protects your OpenFGA server against bursts in traffic.
+- Add a logger interceptor (for example, [in the server configuration](https://pkg.go.dev/github.com/openfga/openfga/server#Config)) that sends logs to your logging platform of choice.
+
+## Database Performance Implication
+
+To ensure good performance for OpenFGA, it is recommended that the [database](./setup-openfga.mdx#configuring-data-storage) be:
+- Used exclusively for OpenFGA and not share with other applications. This allows scaling the OpenFGA database separately and independently.  
+- Indexing to improve performance, especially for check
+
+## Tuples Operation with Authorization Model ID
+
+To ensure reliability while new models are developed, it is <strong>strongly recommended</strong> that 
+- [Check](./perform-check.mdx), [write](./update-tuples.mdx), [expand](../interacting/relationship-queries.mdx#expand) and [list object](../interacting/relationship-queries.mdx#listobjects) be performed against a validated model by specifying the authorization model id 
+- At the same time, updates may be performed on the model and testing may be the new model.
```

Comment:
Not sure this makes sense. 🤔 Are we missing 'performed' between the second `may` and `be`?

Reply:
removed